site/docs/master/namespace.md

```diff
@@ -1,23 +1,23 @@
-# Run in custom namespace
+# Run in a custom namespace
 
-You can run Velero in any namespace.
+The Velero installation and backups by default are run in the `velero` namespace. However, it is possible to customize it.
 
-First, ensure you've [downloaded & extracted the latest release][0].
+### 1) Customize the namespace on the cluster 
 
-Then, install Velero using the `--namespace` flag:
+During [the Velero installation][0], use the `--namespace` flag to inform Velero where to run.
```

Comment:
the `velero install` command needs to be updated to include `--plugins` flag

This is what I use to install velero.
```
velero install  --provider aws \
        --plugins velero/velero-plugin-for-aws:v1.0.0 \
        --bucket $BUCKET \
        --prefix $PREFIX \
        --backup-location-config region=$REGION \
        --snapshot-location-config region=$REGION \
        --secret-file $HOME/.aws/credentials \
        --image $IMAGE
```